__tests__/tests/components/LabelListItem.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { LabelListItem, LabelButton } from 'components';
+
+const defaultProps = {
+  label: 'Awesome Label',
+  removeLabel: () => {},
+};
+
+describe('<LabelListItem />', () => {
+  it('should render component with the correct label', () => {
+    const wrapper = shallow(<LabelListItem {...defaultProps} />);
+    expect(wrapper.find(LabelButton)).toHaveLength(1);
```

Comment:
@MarioArnt can you add a blank line before `expect` calls here and in other places?

Reply:
Done. Could be a good idea to use a linter for the unit tests code style.